daemon/mgr/container_utils.go

```diff
@@ -103,39 +104,46 @@ func (mgr *ContainerManager) generateName(id string) string {
 }
 
 func parseSecurityOpts(meta *ContainerMeta, securityOpts []string) error {
+	var (
+		labelOpts []string
+		err       error
+	)
 	for _, securityOpt := range securityOpts {
 		if securityOpt == "no-new-privileges" {
 			meta.NoNewPrivileges = true
 			continue
 		}
-		if err := parseSecurityOpt(meta, securityOpt); err != nil {
-			return err
+		fields := strings.SplitN(securityOpt, "=", 2)
+		if len(fields) != 2 {
+			return fmt.Errorf("invalid --security-opt %s: must be in format of key=value", securityOpt)
+		}
+		key, value := fields[0], fields[1]
+		switch key {
+		// TODO: handle other security options.
+		case "apparmor":
+			meta.AppArmorProfile = value
+		case "seccomp":
+			meta.SeccompProfile = value
+		case "no-new-privileges":
+			noNewPrivileges, err := strconv.ParseBool(value)
+			if err != nil {
+				return fmt.Errorf("invalid --security-opt: %q", securityOpt)
+			}
+			meta.NoNewPrivileges = noNewPrivileges
+		case "label":
+			labelOpts = append(labelOpts, value)
+		default:
+			return fmt.Errorf("invalid type %s in --security-opt %s: unknown type from apparmor and seccomp", key, securityOpt)
```

Comment:
With the code change, I do not think this line is correct any longer. 

> unknown type from apparmor and seccomp

except `apparmor` and` seccomp`, it seems that we should add `no-new-privileges` and `label`. Right? @YaoZengzeng